model-optimizer/mo/ops/crop.py

```diff
@@ -164,7 +164,7 @@ def _two_inputs_infer(node: Node):
         node['dim'] = dim
         node.out_node().shape = new_shape
 
-        if node.in_node(0).has_valid('value') and not node.graph.graph['cmd_params'].enable_ssd_gluoncv:
+        if node.in_node(0).has_valid('value') and not getattr(node.graph.graph['cmd_params'], 'enable_ssd_gluoncv', False):
```

Comment:
Why did you change this? Is the current logic broken?

Reply:
I don't know why but on Windows I had an error that namespace does not contain attribute `enable_ssd_gluoncv` while on Linux everything was ok, with the same commit. So I added this changes to get the same behavior.

Reply:
Rebased to the latest master, still getting error on Win machine without there changes. While on linux -- everything is ok. @yekruglov, what you think about this?

Reply:
Communicated personally with @yekruglov. On windows I was calling MO by `mo_kaldi.py` so and in that case this variable is not initialized in `cmd_params` namespace. Therefore leaved this safety check for the sake of robustness.